pkgs/applications/audio/youtube-music/default.nix

```diff
@@ -6,27 +6,27 @@
 , stdenv
 , copyDesktopItems
 , nodejs
-, pnpm
+, pnpm_9
```

Comment:
@SuperSandro2000 thoughts on https://github.com/NixOS/nixpkgs/tree/master/pkgs/by-name#changing-implicit-attribute-defaults ? 

because I frequently recommend this during reviews

Reply:
Since youtube-music has a hard requirement on a specific pnpm version through it's lockfile and it took me quite a while to find it but I can move it back.